### PR TITLE
Allow empty string for dns_server param

### PR DESCRIPTION
**Pull request by lzap**
This is my blind attempt to document new behavior introduced by:

https://github.com/theforeman/smart-proxy/pull/602

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -221,7 +221,7 @@
 #
 # $dns_reverse::                DNS reverse zone name
 #
-# $dns_server::                 Address of DNS server to manage
+# $dns_server::                 Address of DNS server to manage, set to empty string to send updates to master server
```
Comment (ekohl) [inline manifests/init.pp]: I think master server is ambiguous and would prefer `set to an empty string to let nsupdate automatically determine the master server`.

File: templates/dns_nsupdate_gss.yml.erb [hunk 2]
```diff
@@ -3,8 +3,10 @@
 # Configuration file for 'nsupdate_gss' dns provider with GSS-TSIG support
 #
 
-# use this setting if you are managing a dns server which is not localhost though this proxy
+# DNS server to talk to, usually localhost but can be arbitrary (remote) host.
```
Comment (ekohl) [inline templates/dns_nsupdate_gss.yml.erb]: `an arbitrary`

File: templates/dns_nsupdate_gss.yml.erb [hunk 3]
```diff
@@ -3,8 +3,10 @@
...
+# This can be also set to blank value to send updates to the master server of the correct zone.
```
Comment (ekohl) [inline templates/dns_nsupdate_gss.yml.erb]: `a blank`

File: manifests/init.pp (hunk 1 above)
Comment (lzap) [reply manifests/init.pp]: Nice thanks I am bad in this.

**Comment by lzap** [comment]
Incorporating your remarks here and in smart proxy repo here is the final version:

```
# DNS server to talk to, usually localhost but can be an arbitrary (remote) host.
# This can be also set to a blank value to let nsupdate to detect master server of a zone.
```

**Pull request closed without merging**